hasura.planx.uk/migrations/1707239828082_update_node_type_names/down.sql

```diff
@@ -0,0 +1,8 @@
+UPDATE
+  public.analytics_logs
+SET
+  node_type = CASE
+    WHEN node_type 'Question' THEN 'Statement'
+    WHEN node_type 'Answer' THEN 'Response'
```

Comment:
```suggestion
    WHEN node_type = 'Question' THEN 'Statement'
    WHEN node_type = 'Answer' THEN 'Response
```

I was seeing an error with the down migration but I think it was just the `=` being missing on these lines. I found it surprisingly hard to spot the difference 😂

Reply:
Ah good catch, this was me being clumsy using `⌘ + D` I think! Fixed here [8b230c9](https://github.com/theopensystemslab/planx-new/pull/2756/commits/8b230c9b43a42818256691a6d8eeed085d66ce8d)